src/main/java/tri/stringreverse/StringReverse.java

```diff
@@ -0,0 +1,16 @@
+package tri.stringreverse;
+
+/**
+ * Output a reversed string.
+ */
+public class StringReverse {                //create class
+    public String reverse(String text){     //function named reverse, the function used a built-in reverse function and return the reversed string
+        return reverse(text);
+    }
+
+    public static void main(String[] args) {
+        StringReverse goal = new StringReverse();
+        System.out.println(goal.reverse());
```

Comment:
I'm not sure which part of my code is the function and which is the main program. Therefore, I just add a random string into the function (I think) to fix that:
System.out.println(goal.reverse()) => System.out.println(goal.reverse("qwerty"));